packages/react-server/core/logging/server.js

```diff
@@ -1,163 +1,163 @@
-var winston = require('winston')
-,   common  = require('./common')
-,   _ = {
-        mapValues     : require("lodash/object/mapValues"),
-        pick          : require("lodash/object/pick"),
-        isPlainObject : require("lodash/lang/isPlainObject"),
-        isEmpty       : require("lodash/lang/isEmpty"),
-        trimLeft      : require("lodash/string/trimLeft"),
-        trunc         : require("lodash/string/trunc"),
-    };
-
-var makeLogger = function(group, opts){
-	var config = common.config[group];
-
-	var fileTransport = new (winston.transports.File)({
-		name      : 'file',
-		level     : config.baseLevel,
-		stream    : process.stdout,
-		json      : false,
-		timestamp : global.TIMESTAMP_REACT_SERVER_LOG_OUTPUT,
+var	winston	=	require('winston')
+,			common		=	require('./common')
+,			_	=	{
+	mapValues					:	require("lodash/object/mapValues"),
+	pick										:	require("lodash/object/pick"),
+	isPlainObject	:	require("lodash/lang/isPlainObject"),
+	isEmpty							:	require("lodash/lang/isEmpty"),
+	trimLeft						:	require("lodash/string/trimLeft"),
+	trunc									:	require("lodash/string/trunc"),
+};
+
+var	makeLogger	=	function(group,	opts){
+	var	config	=	common.config[group];
+
+	var	fileTransport	=	new	(winston.transports.File)({
+		name						:	'file',
+		level					:	config.baseLevel,
+		stream				:	process.stdout,
+		json						:	false,
+		timestamp	:	global.TIMESTAMP_REACT_SERVER_LOG_OUTPUT,
 	});
 
-	var logger = new (winston.Logger)({
-		transports: [
+	var	logger	=	new	(winston.Logger)({
+		transports:	[
```

Comment:
Ah, yeah, I see now.   A bunch of spaces were just changed to tabs.

Let's not have tabs except at the beginning of lines.